CHANGES.md

```diff
@@ -15,6 +15,7 @@ Change Log
 * Fixed documentation for 'Context.drawingBufferHeight' and 'Context.drawingBufferWidth'.
 * Added debug option to 'Scene' to show the depth buffer information for a specified view frustum slice and exposed capability in 'CesiumInspector' widget.
 * Removed `Scene.fxaaOrderIndependentTranslucency`. Use `Scene.fxaa` which is now `true` by default.
+* Added `Scene.pickDepth` to reconstruct the cartesian world position from window coordinates.
```

Comment:
Can we mention that users can now pan/zoom/rotate around 3D models and other opaque geometry?